src/core/Game.ts

```diff
@@ -21,10 +21,10 @@ export class Game {
     /** {@link requestAnimationFrame} request ID to use with {@link cancelAnimationFrame} */
     private animationRequestId: any;
     /** tells the game loop whether its scene needs to be re-rendered (true) or not */
-    private needRender: boolean;
+    public static needRender: boolean;
```

Comment:
why do we mark `needRender` as `public static`?
there should be another way of accessing private properties from an instance of one class in another